apps/www/public/locales/en/common.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
is it possible to generate a json schema for these files? A bit like we do with zod and the designsystemet.config? Would be nice to have something to ensure to locales stay synced schema wise.

Reply:
Possibly, but not something I have looked into